src/xrGame/saved_game_wrapper.cpp

```diff
@@ -88,6 +89,7 @@ CSavedGameWrapper::CSavedGameWrapper(LPCSTR saved_game_name)
         CALifeTimeManager time_manager(alife_section);
         time_manager.load(reader);
         m_game_time = time_manager.game_time();
+        m_valid_save = true;
```

Comment:
Hmm... Does the save is really 'valid' in case of returning from lines 118, 130, 149, 160?

Reply:
Good question. Need @Xottab-DUTY help to clarify is it true or not.